src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -306,11 +306,11 @@ function webDriverValueToBiDi(
   }
 
   if (result.type == 'node') {
-    if (bidiValue.hasOwnProperty('backendNodeId')) {
+    if (Object.prototype.hasOwnProperty.call(bidiValue, 'backendNodeId')) {
```

Comment:
why not https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn ?

Reply:
Sure! My fix was the recommended one in the rule README page, but that should also work.

Reply:
Done.

Except here:

src/bidiMapper/domains/log/logHelper.ts:154:12 - error TS2339: Property 'hasOwn' does not exist on type 'RemoteValue'.
  Property 'hasOwn' does not exist on type 'UndefinedValue'.